components/vf-banner/vf-banner.scss

```diff
@@ -0,0 +1,82 @@
+$vf-data-protection-banner-color--background: set-color(vf-color-gray-dark);
+$vf-color--text: set-color(vf-color-white);
+$vf-data-protection-banner-color--link: inherit;
+
+$vf-data-protection-banner-padding: map-get($vf-spacing-map, vf-spacing-r);
+
+.vf-banner {
+  box-sizing: border-box;
+  padding: 0 12px;
+
+  .vf-tag {
+    margin-right: 16px;
+  }
+}
+
+.vf-banner--fixed {
+  display: grid;
+  left: 0;
+  grid-column: 1 / -1;
+  grid-template-columns:
+    [full-start]
+      minmax(1em, auto)
+      [main-start]
+        minmax(18em, 81.25em) // 1300px
+      [main-end]
+      minmax(1em, auto)
+    [full-end];
+  padding: 0;
+  position: fixed;
+  width: 100%;
+  z-index: 5150;
+}
+
+.vf-banner__content {
+  align-items: center;
+  align-content: center;
+
+  @media (max-width: 63.9375em) {
+    display: block; // overideing the .emlb-grid code
+  }
+
+  @media (min-width: $vf-breakpoint-l) {
+    grid-column: main;
+  }
+}
+
+.vf-banner--top {
+  top: 0;
+}
+
+.vf-banner--bottom {
+  bottom: 0;
+}
+
+.vf-banner--phase {
+  background-color: #FFFADC;
```

Comment:
How do we want to handle this custom colour? Should we just have some bundled vf-core utility colours?

`vf-color-yelllow-light`?

Reply:
if it's to become part of the color palette (in any state) we should be:

- defining them as 'global colours' as design tokens
- making decisions to what's primary/secondary/tertiary etc as design tokens
- getting even granular and defining component tokens in the design tokens

I've yet to start this - for now I've just popped the variable at the top of this component file